Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +10,232 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
```

Comment:
correct, do you mean something like this : 

```
public static bool isValidInput(string[] input)
        {
            if (input == null || input.Length == 0)
            {
                return false; 
            }

            return true; 
        }

        /// <summary>
        /// Return name and balance(current) of person who had the biggest historic balance.
        /// </summary>
        public static string FindHighestBalanceEver(string[] peopleAndBalances)
        {
            if (!isValidInput(peopleAndBalances))
            {
                return "N/A."; 
            }
```

then I see the same code again 3 times. 
and if I only return the `N/A`   and for example a empty string for a valid input I see the same problem. 
So as far as I know , it does not make a difference if I make this a seperate function.

Reply:
Check the method name to make sure that it conforms to C# standard naming conventions and you could maybe look at condensing the if statement with ternary staements. Otherwise, so far so good.

Reply:
oke, but what is then the advantage to do so , instead of doing the check in the function itself. 
I see now the code double. As a function and now as a call. Seems double to me . That is why I asked this

Reply:
solved